src/login/meson.build

```diff
@@ -71,18 +71,19 @@ in_files = [
         ['70-uaccess.rules',   udevrulesdir,  enable_logind and conf.get('HAVE_ACL') == 1],
         ['71-seat.rules',      udevrulesdir,  enable_logind],
         ['73-seat-late.rules', udevrulesdir,  enable_logind],
-        ['systemd-user',       pamconfdir,    enable_logind and pamconfdir != 'no']]
+        ['systemd-user',       pamconfdir,    enable_logind]]
 
 foreach tuple : in_files
         file = tuple[0]
+	dir = tuple[1]
         custom_target(
                 file,
                 input : file + '.in',
                 output: file,
                 command : [meson_render_jinja2, config_h, '@INPUT@'],
                 capture : true,
-                install : tuple[2],
-                install_dir : tuple[1])
+                install : tuple[2] and ((dir == pkgsysconfdir) ? install_sysconfdir_samples : (dir != 'no')),
```

Comment:
Hmm, maybe define a variable for the part after `and` (since it needs a revision because of the indentation anyway…) ?